### PR TITLE
Add Hotjar to Docs site

### PR DESCRIPTION
**Pull request by PrajwalPrashanth**
Verified from hotjar
image
https://prajwalprashanth-demo.readthedocs.io/en/pp-ch2556-hotjar-for-docs/

### CODE REVIEW

File: docs/_templates/layout.html [hunk 1]
```diff
@@ -1,33 +1,39 @@
...
+    r = o.createElement('script'); r.async = 1;
+    r.src = t + h._hjSettings.hjid + j + h._hjSettings.hjsv;
+    a.appendChild(r);
+  })(window, document, 'https://static.hotjar.com/c/hotjar-', '.js?sv=');
+</script>
+
+{% endblock %}
+
 {% block extrabody %}
-  <nav
```
Comment (sidujjain) [inline docs/_templates/layout.html:4]: indentation changes can be reversed

File: docs/_templates/layout.html [hunk 2]
```diff
@@ -1,33 +1,39 @@
...
+{{ super() }}
 {% endblock %}
 
 {% block mobile_nav%} {% endblock %}
 
 {% block sidebartitle %}
-  {% include "searchbox.html" %}
-{% endblock %}
+{% include "searchbox.html" %}
+{% endblock %}
```
Comment (sidujjain) [inline docs/_templates/layout.html:39]: nit: newline

File: docs/_templates/layout.html [hunk 3]
```diff
@@ -1,33 +1,39 @@
 {% extends '!layout.html' %}
 
+{% block extrahead %}
+<script>
+  (function (h, o, t, j, a, r) {
+    h.hj = h.hj || function () { (h.hj.q = h.hj.q || []).push(arguments) };
+    h._hjSettings = { hjid: 1819338, hjsv: 6 };
```
Comment (sidujjain) [inline docs/_templates/layout.html:7]: can this id come from env? 

this section can probably go to footer as we don't want to block content rendering on this.

File: docs/_templates/layout.html (hunk 2 above)
Comment (sidujjain) [inline docs/_templates/layout.html:39]: nit: newline

File: docs/_templates/layout.html (hunk 3 above)
Comment (PrajwalPrashanth) [reply docs/_templates/layout.html:7]: I'll check out getting the id from env.

This can stay in head, hotjar docs says its async.

Comment (sidujjain) [reply docs/_templates/layout.html]: it will still increase page load time because resources are limited on devices, especially mobile.

Comment (PrajwalPrashanth) [reply docs/_templates/layout.html]: Moved to footer 👍

**Pull request merged**